server/app/auth/CiviFormProfileMerger.java

```diff
@@ -0,0 +1,69 @@
+package auth;
+
+import java.util.Optional;
+import java.util.function.BiFunction;
+import javax.inject.Provider;
+import models.Account;
+import models.Applicant;
+import org.pac4j.core.profile.UserProfile;
+import repository.UserRepository;
+
+/** This class serves to abstract out common user profile merging logic into a shared helper. */
```

Comment:
Similar to the below comment, use a sentence fragment.


 \* Common {@code UserProfile} merging logic.